### DISCUSSION TITLE
Which city is home to the iconic statue, Christ the Redeemer?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#578] [Galaxy Trivia] [answered]
**Bonus:** How high is it?

**Comment by alejandroch1202** [comment, marked as answer]
- Rio de Janeiro, Brazil.
- 98 feet (30 metres).

**Comment by thuongtruong1009** [comment]
The iconic statue of Christ the Redeemer is located in the city of Rio de Janeiro, Brazil. The statue is located on top of Corcovado Mountain, which overlooks the city.

The statue itself is 30 meters (98 feet) tall and its arms span 28 meters (92 feet) wide, making it one of the largest art deco statues in the world. The pedestal on which the statue stands is 8 meters (26 feet) tall, bringing the total height of the monument to 38 meters (125 feet).
